blog-posts/common-approaches-to-handling-subscriptions/common-approaches-to-handling-subscriptions.md

```diff
@@ -0,0 +1,178 @@
+---
+published: true
+title: "Common Approaches to Handling Subscriptions"
+cover_image: ""
+description: "This blog article covers the some common approaches to handling Observable subscriptions with RxJS."
```

Comment:
I'm not that good in english language, but `the some common` sounds weird to me :D